data/user-environment-generators/61-eos-kolibri.sh.in

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
-XDG_DATA_DIRS="@KOLIBRI_DATA_DIR@/content/xdg/share:${XDG_DATA_DIRS:-/usr/local/share:/usr/share}"
-echo "XDG_DATA_DIRS=$XDG_DATA_DIRS"
+if flatpak info --system --show-ref @KOLIBRI_FLATPAK_ID@ 2>/dev/null; then
```

Comment:
The conditional looks good to me. I wonder about the `--system` flag. I guess it's fine since all flatpaks in Endless OS are system-wide.

Reply:
Thanks for checking :) Yep, that's intentional. The reason for it is the system-wide Kolibri daemon is only going to run if Kolibri is installed on the system. It would be really weird if Kolibri ended up in the user installation instead of the system installation, but if that _did_ happen, it shouldn't do any of these system integration things.